### PR TITLE
Happy-thoughts-api-Disa

### PR DESCRIPTION
**Pull request by Disa-san**
I really enjoyed working on this project and I think it is great to see your frontend and backend working together. I still struggle generally with the syntax, but I am trying to give myself a break and think that with time it will come to me. No stretch goals, as I was just happy to get it all together. 
 

On a general note, I think it has been greet to have codealongs with both Jenny and Van (on this occasion) . Just hearing two different people say the same thing in different ways helps with the learning, I believe.

### CODE REVIEW

**Comment by Disa-san** [comment]
Frontend: https://happy-thoughts-project-disa.netlify.com/
Backend: https://project-happy-api.herokuapp.com/

File: server.js [hunk 1]
```diff
@@ -19,11 +36,50 @@ app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// app.get('/', (req, res) => {
+//   res.send('Hello world')
+// })
```
Comment (JennieDalgren) [inline server.js]: remove this

File: server.js [hunk 2]
```diff
@@ -7,6 +7,23 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+  hearts: {
+    type: Number,
+    required: true
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now
+  }
+})
+
```
Comment (JennieDalgren) [inline server.js]: ⭐️

File: server.js [hunk 3]
```diff
@@ -19,11 +36,50 @@ app.use(cors())
 app.use(bodyParser.json())
```
Comment (JennieDalgren) [inline server.js]: Also add the error handling for the database connection. 
Check Damiens video about setting that middleware up here:
https://technigo.wistia.com/medias/l5wqrcu4to?wtime=9m22s

**Pull request closed without merging**